registrations/api.py

```diff
@@ -284,9 +314,12 @@ def filter_queryset(self, queryset):
                 registrations.append(registration)
             queryset = queryset.filter(registration__in=registrations)
         elif self.action == "list":
```

Comment:
Also here: perhaps superusers should be allowed to perform the `list` action?

Reply:
Allowed superusers to have access in https://github.com/City-of-Helsinki/linkedevents/pull/631/commits/2a60a351dca92e2849af5afd8ace752c381dda44